sync-data/main.go

```diff
@@ -156,11 +156,14 @@ func reuploadData(ctx context.Context, logger logging.Logger, destination DataDe
 	coll := mongoClient.Database(QueryableTabularDatabaseName).Collection(QueryableTabularCollectionName)
 	// My brutal index to speed up re-checking
 	// With this index, this func takes ~10 seconds locally. Without this, this func can take >10 minutes
-	indexName := "viam-teleop-tools-time-received-index"
+	indexName := "viam-teleop-tools-time-received-index-part-time"
```

Comment:
Doesn't matter what this is named -- just needs to be unique. Now that you've all run this locally, adding new chars to the end is easier than having everyone delete and recreate the index.